src/content/docs/features/single-instance.mdx

```diff
@@ -1,14 +1,76 @@
 ---
 title: Single Instance
-description: Ensure a single instance of your tauri app is running.
+description: Plugin to ensure that a single instance of your tauri app is running at a time.
 ---
 
-import Stub from '@components/Stub.astro';
 import PluginLinks from '@components/PluginLinks.astro';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 <PluginLinks plugin="single-instance" showJsLinks={false} />
 
-<Stub>
-	Based on
-	https://github.com/tauri-apps/plugins-workspace/tree/v2/plugins/single-instance
-</Stub>
+Plugin to ensure that a single instance of your tauri app is running at a time.
+
+
+## Install
+
+_This plugin requires a Rust version of at least **1.75**_
+
+You can install the Core plugin by adding the following to your `Cargo.toml` file:
+
+<Tabs>
+  <TabItem label="crates.io">
+```toml title="src-tauri/Cargo.toml" ins={2}
+[dependencies]
+tauri-plugin-single-instance = "2.0.0-beta"
+```
+  </TabItem>
+  <TabItem label="Git">
+
+```toml title="src-tauri/Cargo.toml" ins={2}
+[dependencies]
+tauri-plugin-single-instance = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v2" }
+```
+  </TabItem>
+</Tabs>
+
+## Usage
+
+To use this plugin, you need to register the core plugin with Tauri. The plugin init() method takes a closure that is called when a new app instance was started, but closed by the plugin. The closure has three arguments: the AppHandle, the list of arguments (important for instance on deep link usage) and the current working directory:
```

Comment:
Could you reword a bit this section to make it clearer? Imagine than non Rust (or beginner level) devs will eventually read this. Also, originally the features guides in general are to be recipes (doesn't help that the name we use internally is 'guide') https://github.com/tauri-apps/tauri-docs/blob/next/.github/CONTRIBUTING.md#recipe. I asked on discord for clarification, but I believe it should so.

Feel free to play around, just throwing out ideas to improve.

To follow the docs pattern, what do you thing of adding a Permission section (as other guides have) but say that it doesn't need any setup or something along this lines?